terminaltables/tables.py

```diff
@@ -92,3 +92,66 @@ class DoubleTable(WindowsTableDouble):
     """Cross-platform table with box-drawing characters. On Windows it's double borders, on Linux/OSX it's unicode."""
 
     pass
+
+
+class GithubFlavoredMarkdownTable(BaseTable):
+    """Github flavored markdown table.
+
+    https://help.github.com/articles/github-flavored-markdown/#tables
+    """
+
+    CHAR_VERTICAL = '|'
+    CHAR_HORIZONTAL = '-'
+
+    def __init__(self, table_data):
```

Comment:
@Robpol86 CI warns for this issue: `W:106, 4: __init__ method from base class 'BaseTable' is not called (super-init-not-called)`, but I think there is no need to call the super here.